HW Assignment #1/About_Me.html

```diff
@@ -0,0 +1,26 @@
+
+
+
+<link rel="stylesheet" type="text/css" href="style.css">
+
+<center>
+	<h1> TASNEEM KARIM</h1>
+
+
```

Comment:
Center tags are not supported in HTML5. Try doing text alignments in your css file instead. http://www.w3schools.com/css/css_text.asp